test/List.js

```diff
@@ -67,6 +69,10 @@ List.prototype[FL.chain] = function(f) {
     Z.concat(f(this.head), Z.chain(f, this.tail));
 };
 
+List.prototype[FL.alt] = function(other) {
```

Comment:
Good catch. I find this counter-intuitive, so I tend to forget. Thanks for the reminder. :)